### PR TITLE
Added placeholders and translations for discrepencies between EN and CN -> CN and EN

### PR DESCRIPTION
**Pull request by stoiji**
###### Description of changes

Missing files added, and translations in places a base language was available.

###### Review guide

Look for files translation content in CN to make sure it is readable because it came from EN as base.

To review this PR locally, please run the following commands:

It's the same as original repo readme.md. yarn install, yarn build, and yarn start

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const config = {
   url: 'https://docs.radxa.com',
   baseUrl: '/',
   trailingSlash: false,
-  onBrokenLinks: 'throw',
```
Comment (RadxaYuntian) [inline docusaurus.config.js:14]: This is a safety measure. We want the broken links to break the build so we never merge broken PR to the repo.

As for the generated `missing` file you can add it to `.gitignore`.

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]

**Comment by RadxaYuntian** [comment]
BTW you don't need to edit the `review guide` section in PR message. It is a generic instruction for how to checkout PR and run it locally.

File: docusaurus.config.js (hunk 1 above)
Comment (stoiji) [reply docusaurus.config.js:14]: I got broken links when building, did not view where. I just disabled this.

**Review by RadxaYuntian — DISMISSED** [review]
We can fix those translation issues later.

**Review by RadxaYuntian — APPROVED** [review]

**Pull request merged**